[ERROR] No .editorconfig properties applicable for files in a Maven module without .editorconfig

When building in a module directory of a multimodule Maven project, the `.editorconfig` file in the root of the source tree is not seen. Something like the following is logged:

```
[ERROR] No .editorconfig properties applicable for files under ...
```

--- a/editorconfig-maven-plugin/src/main/java/org/ec4j/maven/AbstractEditorconfigMojo.java
+++ b/editorconfig-maven-plugin/src/main/java/org/ec4j/maven/AbstractEditorconfigMojo.java
@@ -31,7 +31,6 @@ import org.apache.maven.plugins.annotations.Parameter;
 import org.codehaus.plexus.util.DirectoryScanner;
 import org.ec4j.core.Cache.Caches;
 import org.ec4j.core.Resource.Resources;
-import org.ec4j.core.ResourcePath.ResourcePaths;
 import org.ec4j.core.ResourceProperties;
 import org.ec4j.core.ResourcePropertiesService;
 import org.ec4j.core.model.PropertyType;
@@ -196,11 +195,9 @@ public abstract class AbstractEditorconfigMojo extends AbstractMojo {
 
         try {
             final ViolationHandler handler = createHandler();
-            final Path sourceTreeRoot = Paths.get(System.getProperty("maven.multiModuleProjectDirectory"));
 
             final ResourcePropertiesService resourcePropertiesService = ResourcePropertiesService.builder() //
                     .cache(Caches.permanent()) //
-                    .rootDirectory(ResourcePaths.ofPath(sourceTreeRoot, charset)) //
                     .build();
             handler.startFiles();
             boolean propertyMatched = false;

